src/utils/vee-validate.js

```diff
@@ -118,6 +118,15 @@ extend('isPhoneNumber', {
   }
 })
 
+extend('notMinus', {
+  message: (_, values) => i18n.t('errors.not_minus', values),
+  validate: value => {
+    const t = true
+    const f = false
+    return value >= 0 ? t : f
```

Comment:
sama aja boong. Kenapa `true` nya ga langsung aja.

coba kalau gini jalan ga

```suggestion
    return value >= 0
```

Reply:
ga bisa pak error kalau ga pake variable
![image](https://user-images.githubusercontent.com/15316339/79849485-bfb43400-83ec-11ea-9ceb-be4701383beb.png)